indico/htdocs/js/indico/modules/registration/reglists.js

```diff
@@ -247,7 +261,22 @@
         });
 
         $('.reglist-filter input:checkbox').on('change', function() {
-            colorizeFilter($(this).closest('.filter'));
+            colorizeFilter($(this).closest('.dropdown').siblings('.filter'));
+        });
+
+        $('.reglist-filter .title').on('mouseover', function() {
+            var title = $(this);
+            if (this.offsetWidth < this.scrollWidth) {
+                var qtip = title.qtip({hide: 'mouseout', content: title.text(), overwrite: false}).qtip('show');
+            }
+        });
+
+        $('#report-filter-select-all').on('click', function () {
+            $('.report-filter-dialog .report-column.striped .icon-eye').click();
```

Comment:
I also wouldn't use an icon class to select items - that's a pure style thing, and changing the icon in the future shouldn't break functionality. I'd add a `js-whatever` class and use that instead